### ISSUE TITLE
Cannot save councillor reply with special characters

### ISSUE THREAD

**Issue by henare** [openaustralia/planningalerts#1026]
[PlanningAlerts/production] ActiveRecord::StatementInvalid: Mysql2::Error: Incorrect string value: '\xEF\x81\x90 Pl...' for column 'text' at row...[snip]

From reply: http://planningalerts.writeit.ciudadanointeligente.org/en/thread/planning-application-at-39-barns-lane-coolum-beach-qld-7/

## Backtrace

    line 83 of [PROJECT_ROOT]/app/models/comment.rb: block in create_replies_from_writeit!
    line 81 of [PROJECT_ROOT]/app/models/comment.rb: collect
    line 81 of [PROJECT_ROOT]/app/models/comment.rb: create_replies_from_writeit!

[View full backtrace and more info at honeybadger.io](https://app.honeybadger.io/projects/37075/faults/30808887)

**Comment by henare** [comment]
This is because the column in question is not UTF-8.

In the past [we already](https://github.com/openaustralia/planningalerts/commit/3e926d0faca9523d6488525fe110c4866646c6ae) changed all tables to be UTF-8 but it seems like the server is not set to default to UTF-8. This seems to mean that as we've created new tables they are defaulting to `latin1` instead of UTF-8. Great.

I think the way to fix this is to rerun that migration on the server and set the server to use UTF-8 by default.

**Comment by henare** [comment]
I've added this below the `[mysqld]` section in `/etc/mysql/my.cnf` and restarted MySQL:

```
character-set-server=utf8
collation-server=utf8_general_ci
```

I couldn't find a neat way of running that migration again so I've created a new migration instead.

I've deployed that to staging and the migration took about 3 minutes. I'd assume that migration would block tables as it runs so I'm not sure it's a great idea running it during the day.

After we run it in production we also need to reload the reply that was affected by this bug.
